generator/altitude_generator.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "geometry/point2d.hpp"
+
+#include "indexer/feature_altitude.hpp"
+
+#include "std/string.hpp"
+
+namespace routing
+{
+class IAltitudeGetter
```

Comment:
We don't, it's a bad practice. Are you ready to rename IAltitudeGetter when it'll have non-virtual methods or fields? Is it interesting for readers that IAltitudeGetter is a pure interface?